### PR TITLE
add coder to the Open a Remote Window list

### LINKED ISSUE #93: Display the Coder workspaces here in this list? Or at least an option to list the workspaces?
See attached video

https://user-images.githubusercontent.com/10648092/235918497-41e76143-ecfb-4c60-b6ff-496b1d24fab8.mp4

### PR DESCRIPTION
**Pull request by rodrimaia**
Closes #93 

We added  Coder actions to the "Open a Remote Window" menu list. 

image

### CODE REVIEW

File: src/workspacesProvider.ts [hunk 1]
```diff
@@ -106,7 +106,9 @@ class AgentMetadataTreeItem extends vscode.TreeItem {
       metadataEvent.description.display_name.trim() + ": " + metadataEvent.result.value.replace(/\n/g, "").trim()
 
     super(label, vscode.TreeItemCollapsibleState.None)
-    this.tooltip = "Collected at " + metadataEvent.result.collected_at
+    const collected_at = new Date(metadataEvent.result.collected_at).toLocaleString()
```
Comment (rodrimaia) [inline src/workspacesProvider.ts:109]: squeezing this in 😄  to improve sidebar metadata tooltip

**Comment by matifali** [comment]
Maybe you should remove the preceding Coder from
Coder: Open Workspace

**Review by matifali — COMMENTED** [review]
Maybe you should remove the preceding Coder from
`Coder: Open Workspace` --> `Open Workspace`
To make it consistent wit other options in the list

**Review by matifali — APPROVED** [review]
If you can address the comment, except that it all looks perfect.

**Pull request merged**